lib/FieldType/DataTransformer/MultipleCountryValueTransformer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Country\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * DataTransformer for Country\Value to be used with form type handling multiple selections.
+ * Needed to display the form field correctly and transform it back to an appropriate value object.
+ */
+class MultipleCountryValueTransformer implements DataTransformerInterface
+{
+    /**
+     * @var array Array of countries from ezpublish.fieldType.ezcountry.data
```

Comment:
I wouldn't specify where the value is taken from. The service container already has that info, and it kind of adds a two-way relationship between the class and the service definition.